### PR TITLE
Create spark-standalone framework

### PR DESCRIPTION
**Pull request by mrbrowning**

### CODE REVIEW

File: frameworks/spark/universe/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "packagingVersion": "2.0",
+  "name": "spark",
+  "version": "{{package-version}}",
+  "maintainer": "support@mesosphere.io",
+  "description": "An spark implementation of a service using the DC/OS SDK",
```
Comment (gabrielhartmann) [inline frameworks/spark/universe/package.json]: `A Spark...`

File: frameworks/spark/universe/package.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "selected": false,
+  "framework": true,
+  "tags": ["spark"],
+  "postInstallNotes": "DC/OS spark is being installed!\n\n\tDocumentation: https://github.com/mesosphere/dcos-commons/tree/master/frameworks/spark\n\tIssues: https://docs.mesosphere.com/support/",
```
Comment (gabrielhartmann) [inline frameworks/spark/universe/package.json]: `Spark` should be capitalized here and below I believe.  Is `DC/OS Spark` accurate?  I think ASF is picky about Spark being called `Apache Spark`.  Maybe `DC/OS Apache Spark`.

File: frameworks/spark/universe/config.json [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  "type":"object",
+    "properties":{
+      "service":{
+        "type":"object",
+        "description": "DC/OS service configuration properties",
+        "properties":{
+          "name" : {
+            "description":"The name of the service instance",
+            "type":"string",
+            "default":"spark"
```
Comment (gabrielhartmann) [inline frameworks/spark/universe/config.json]: Probably don't want to take the name `spark`.  `spark-standalone`?

File: frameworks/spark/src/main/dist/svc.yml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    tasks:
+      server:
+        goal: RUNNING
+        cmd: "./bootstrap && $MESOS_SANDBOX/{{SPARK_VERSION}}/sbin/start-master.sh -p {{SPARK_MASTER_PORT}} --webui-port {{SPARK_MASTER_WEBUI_PORT}}"
+        cpus: {{DAEMON_CPUS}}
+        memory: {{DAEMON_MEM}}
+        ports:
+          data:
+            port: {{SPARK_MASTER_PORT}}
+          webui:
```
Comment (gabrielhartmann) [inline frameworks/spark/src/main/dist/svc.yml]: No proxylite or anything?

File: frameworks/spark/README.md [hunk 5]
```diff
@@ -0,0 +1 @@
+Template README
```
Comment (gabrielhartmann) [inline frameworks/spark/README.md]: Should replace this with something better.  Doesn't have to be perfect, but `Template README` should be replaced.

File: frameworks/spark/universe/package.json (hunk 2 above)
Comment (keithchambers) [inline frameworks/spark/universe/package.json]: I recommend we call it "Apache Spark Standalone".

File: frameworks/spark/universe/resource.json [hunk 6]
```diff
@@ -0,0 +1,17 @@
+{
+  "assets": {
+    "uris": {
+      "jre-tar-gz": "https://downloads.mesosphere.com/dcos-commons/artifacts/jre-8u91-linux-x64.tar.gz",
```
Comment (vishnu2kmohan) [inline frameworks/spark/universe/resource.json]: jre-8u121

**Comment by mrbrowning** [comment]
@mesosphere-ci retest this please

**Pull request merged**